### PR TITLE
Fix the `contao-setup` command

### LINKED ISSUE #7354: Fresh installation of 5.4 not possible
### Affected version(s)

5.4.x-dev

### Description

Unfortunately, the changes in https://github.com/contao/contao/commit/ce48ebc5f69ee658f6ef357fc3c87fd2dbc36f79 seem to break a new installation. `contao:symlinks` tries to create symlinks to paths that have not yet been created in the `contao:install` command now (/system/modules/ in my case).

Related: https://github.com/contao/contao/pull/7094

### PR DESCRIPTION
**Pull request by leofeyer**
Fixes #7354

### CODE REVIEW

**Review by aschempp — CHANGES_REQUESTED** [review]
I think the easiest and best solution is to remove `assets/css` and `assets/js` from the `contao:install` command. They are totally unnecessary, because they will be created on the fly when needed. This would make the asset-dir handling obsolete in the install command, and only make it handled by the symlink command (which is already implemented correctly imho).

**Comment by aschempp** [comment]
Also need to remove `$this->addEmptyDir($this->imageDir)` since the images dir is inside assets. I verified an default Contao 5.4 with demo theme and the images dir is created on the fly as well.

File: core-bundle/src/Command/InstallCommand.php [hunk 1]
```diff
@@ -96,4 +104,21 @@ private function addEmptyDir(string $path): void
...
+            return null;
+        }
+
+        $composerConfig = json_decode(file_get_contents($composerJsonFilePath), true, 512, JSON_THROW_ON_ERROR);
+
+        if (null === ($componentDir = $composerConfig['extra']['contao-component-dir'] ?? null)) {
+            return null;
+        }
+
+        return $componentDir;
```
Comment (ausi) [inline core-bundle/src/Command/InstallCommand.php]:
```suggestion
        return $composerConfig['extra']['contao-component-dir'] ?? null;
```

**Review by ausi — DISMISSED** [review]

**Comment by leofeyer** [comment]
@aschempp Changed in 8c0a770bce935170f3338df31af24fe4ede98956.

File: core-bundle/src/Command/InstallCommand.php (hunk 1 above)
Comment (leofeyer) [reply core-bundle/src/Command/InstallCommand.php]: Changed in 8c0a770bce935170f3338df31af24fe4ede98956.

**Review by ausi — APPROVED** [review]
I’m not 100% sure that there will be no issues if the empty directories are missing. But if a bug appears because of that I’d fix it where it is caused anyways, so I’m fine with not creating the empty assets directories. 👍

**Review by aschempp — APPROVED** [review]

**Pull request merged**